valohai/internals/utils.py

```diff
@@ -18,3 +18,9 @@ def get_sha256_hash(filepath: str) -> str:
 
     with open(filepath, "rb") as f:
         return get_fp_sha256(f)  # type: ignore
+
+
+def is_local_file_path(path_str: str) -> bool:
+    if "://" in path_str:
+        return False
+    return True
```

Comment:
**_what!_**

no `file://` allowed!?

... is probably fine :rofl: leaving `file://` as an escape from the ignoring behavior can be useful at some point :laughing:

Reply:
well...`file://` is technically a local file system so I guess it counts 🤔 but on the other hand, it's true if it's called in the container it would point to the container's file system, so yeah, maybe we can escape it

Reply:
it's fine as-is :+1: